deployment-server/README.md

```diff
@@ -0,0 +1,22 @@
+# deployment-server
+
+The deployment server listens for the github `pull_request` web hook, if the event has the `merged_at` attribute set then it will `nixops deploy` the `origin/master` branch from where it is running.
+
+Note that for simplicity it will exclude the `nixops` machine so if you make changes to that machine configuration you will need to deploy manually. One of the issues is that if a change is made to the deployment server service then it will restart _during_ the deployment and end the deployment early.
+
+## Deployment
+
+The deployment server is run on the nixops server however it is only enabled if the `githubWebhookKey` is present in `secrets.json`.
+
+The deployment server relies on a nixops deployment called `playgrounds` existing. The reason for this is that it's difficult to manage ssh keys for the service however when a deployment is created the keys it is created with are stored in the state. Additionally this means you can rollback the deployment manually etc. Hopefully you called your original deployment `playgrounds` however if you didn't you can just create a new deployment with that name and delete the old one with the `--force` flag set.
```

Comment:
Make the name configurable? Maybe `playgrounds-dev`?

Reply:
I could do but I would end up passing it all the way through the nixops stuff and out into an additional configuration parameter you would need to set. Seems a bit over the top when you can just rename your deployment. Also bear in mind that it seems unlikely that this will be used in anything other than the alpha environment.

Reply:
ok I've moved it up to `default.nix`. We may decide in the future that we want some sort of optional overrides file or something but until someone actually has a use case for changing these variables I think it's fine to leave them at this top level.